docs/chapters/baseplatform/building-PELUX-sources.rst

```diff
@@ -74,17 +74,16 @@ Intel NUC are ``.wic`` images, which are complete disk-images. For the Raspberry
 Pi 3, the preferred format is ``.rpi-sdimg`` which can be directly written to
 the SD card.
 
-Building with Vagrant
----------------------
+Building with Docker
+--------------------
 
-When we build internally at Pelagicore in our CI system, we use vagrant.  Please
+When we build internally at Pelagicore in our CI system, we use Docker.  Please
 note that we only run this setup in a GNU/Linux system at Pelagicore. It should
```

Comment:
This line is a bit repetitive. I would suggest taking the opportunity to remove the seconds line's "at Pelagicore".

Reply:
Fixed!